src/btree/bt_ret.c

```diff
@@ -116,6 +126,12 @@ __wt_value_return_buf(WT_CURSOR_BTREE *cbt, WT_REF *ref, WT_ITEM *buf)
         /* Take the value from the original page cell. */
         cell = WT_COL_PTR(page, &page->pg_var[cbt->slot]);
         __wt_cell_unpack(session, page, cell, &unpack);
+        if (start != NULL && stop != NULL) {
```

Comment:
Is it worth turning this into a macro for less code duplication?

Reply:
This code will likely change when we fix search as we'll need to set the time pairs from more spots (not using a `WT_UNPACK`). I'd prefer to wait until then to figure out the best way to separate this code out if that's fine with you.

Reply:
fbm.